app/views/works/work_file_list_show.html.erb

```diff
@@ -23,7 +23,13 @@
       <div class="show-sub-head-body request-section">
         <div class="pr-2 long-text-line-height">
         <div class="pb-sm-3 pl-sm-3 text-nowrap float-sm-right">
-          <%= link_to "Request Access", "javascript: alert('under construction, not yet implemented')", class: "btn btn-emphasis mb-2" %>
+          <%= link_to "Request Access",
+            request_oral_history_access_form_path(@work.friendlier_id),
+            class: "btn btn-emphasis mb-2",
+            analytics_category: 'Work',
+            analytics_action: "show_restricted_oral_history_request_form",
+            analytics_label: @work.friendlier_id
+          %>
```

Comment:
Let's confirm that stakeholders want this tracked in google analytics before tracking it. 

Do we want to track when the user just *displays the form* without submitting it? Which is what's being done here? Or instead, track when the user submits the form? Or track both actually, I guess would let us see if people ask for the form but don't submit it?  Or maybe we don't actually care about any of this and don't need to track anything, we don't need to make extra work for ourselves if our stakeholders don't want/need it.

Reply:
Fair. I can simply remove. We are tracking actual submitted requests after all so this is probably overkill.

Reply:
Oh, I guess actual submission of form is tracked in the db, no need to track it in GA, right!

I'm not sure we need to track every time someone asks for the form to be displayed (and may or may not submit it) though, unless this is something stakeholders brought up?